packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -688,7 +688,12 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
-  bool _active = false;
+  RenderObjectOwner _owner;
+  RenderObjectOwner get owner => _owner;
+  void set owner(RenderObjectOwner value) {
```

Comment:
As it turns out, there's only one place where the owner of an element object is used, and that's in the assert in GestureDetector.dart. I'm sort of on the fence about adding the owner field to Element at all.